qiskit/pulse/macros.py

```diff
@@ -104,3 +112,173 @@ def measure_all(backend) -> Schedule:
         A schedule corresponding to the inputs provided.
     """
     return measure(qubits=list(range(backend.configuration().n_qubits)), backend=backend)
+
+
+def chunking_gaussian_square(
+    duration: int,
+    amp: ParameterValueType,
+    angle: ParameterValueType,
+    sigma: ParameterValueType,
+    risefall_sigma_ratio: ParameterValueType,
+    channel: PulseChannel,
+    granularity: int,
+    name: Optional[str] = None,
+    limit_amplitude: Optional[bool] = None,
+    chunk_size: int = 256,
+    min_chunk_number: int = 10,
```

Comment:
I wonder if the default should be 1? Even if there are only a few chunks in some pulses, if you are doing a scan where some pulses end up chunked, it would be more efficient in terms of waveform memory for the shorter pulses to be chunked as well so that they share the rise/fall chunks.